ui/src/app/components/login-gov-ial2-notification.tsx

```diff
@@ -69,7 +70,7 @@ const LoginGovIAL2Notification = () => {
     <AlarmExclamation style={styles.icon}/>
     <div style={styles.text}>Please verify your identity by 10/06/2021.
     </div>
-    <Button type='primary' style={styles.button} onClick={() => location.replace('/?workbenchAccessTasks=true')}>
+    <Button type='primary' style={styles.button} onClick={() => navigate(['data-access-requirements'])}>
```

Comment:
since the DAR is not the homepage, this is now a simple navigation without reload.